shared/src/commonMain/kotlin/org/hyperskill/app/step_quiz/presentation/StepQuizReducer.kt

```diff
@@ -575,6 +575,7 @@ internal class StepQuizReducer(
             else -> emptySet()
         }
 
+    @Suppress("ReturnCount")
```

Comment:
In my opinion, it's better to suppress this issue in the baseline. `Suppress` should be used in case you wanna leave the warned code as is.